crates/biome_js_analyze/src/lint/style/no_restricted_imports.rs

```diff
@@ -315,6 +332,93 @@ declare_lint_rule! {
     /// ```js,use_options
     /// import { Bar } from 'import-bar';
     /// ```
+    ///
+    /// ### `patterns`
+    ///
+    /// This option allows you to specify multiple modules to restrict using gitignore-style patterns.
```

Comment:
```suggestion
    /// ### `patterns`
    /// **Since `v2.1.0`**
    ///
    /// This option allows you to specify multiple modules to restrict using gitignore-style patterns.
```